apps/easypid/src/features/receive/slides/CredentialCardSlide.tsx

```diff
@@ -1,46 +1,64 @@
 import type { CredentialDisplay } from '@package/agent'
-import { AnimatedStack, Heading, Stack, YStack, useInitialRender, useSpringify } from '@package/ui'
-import { DualResponseButtons, FunkeCredentialCard, useWizard } from 'packages/app/src'
-import { FadeIn, LinearTransition } from 'react-native-reanimated'
+import { Heading, MiniCardRowItem, Paragraph, Stack, YStack } from '@package/ui'
+import { DualResponseButtons, useHasInternetConnection, useWizard } from 'packages/app/src'
 
 interface CredentialCardSlideProps {
+  type: 'presentation' | 'pin' | 'noAuth'
   display: CredentialDisplay
-  onContinue: () => void
 }
 
-export const CredentialCardSlide = ({ display, onContinue }: CredentialCardSlideProps) => {
+const getContentType = (type: 'presentation' | 'pin' | 'noAuth', issuerName: string) => {
+  switch (type) {
+    case 'presentation':
+      return {
+        title: 'Share who you are',
+        subtitle: `To receive this card from ${issuerName}, you need to share your identity.`,
+      }
+    case 'pin':
+      return {
+        title: 'Card offered',
+        subtitle: `To receive this card from ${issuerName}, you need to enter your PIN.`,
+      }
+    default:
+      return {
+        title: 'Card offered',
+        subtitle: `${issuerName} wants to give you the following card:`,
```

Comment:
give is maybe not hte right word, is issue too technical?

```suggestion
        subtitle: `${issuerName} wants to issue you the following card:`,
```